packages/dom-adapters/src/caret/utils/useSelectionChange.ts

```diff
@@ -52,6 +52,11 @@ export const useSelectionChange = createSingleton(() => {
       return false;
     }
 
+    if (selection.rangeCount <= 0) {
+      // special case for native input elements
+      return false;
+    }
```

Comment:
1) Please use block-style comment
2) Put it above to related code 
3) Explain more details: why it was added, which case you're mentioning